OWNERS

```diff
@@ -0,0 +1,5 @@
+assignees:
+- brendandburns
```

Comment:
By throughout, do you mean that they should be in all of the `assignees` sections, or the directory OWNERS files?